lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -5043,23 +5043,12 @@ instance IsServerError ErrAssignRedeemers where
                 , "for one of your redeemers since I am unable to decode it"
                 , "into a valid Plutus data:", pretty r <> "."
                 ]
-        ErrAssignRedeemersUnresolvedTxIns ins ->
-            -- Note that although this error is thrown from
-            -- '_assignScriptRedeemers', it's more related to balanceTransaction
-            -- in general than to assigning redeemers. Hence we don't mention
-            -- redeemers in the message.
-            apiError err400 UnresolvedInputs $ T.unwords
-                [ "The transaction I was given contains inputs I don't know"
-                , "about. Please ensure all foreign inputs are specified as "
-                , "part of the API request. The unknown inputs are:\n\n"
-                , pretty ins
-                ]
-        ErrAssignRedeemersTranslationError TimeTranslationPastHorizon ->
+        ErrAssignRedeemersTranslationError (TimeTranslationPastHorizon t) ->
             -- We differentiate this from @TranslationError@ for partial API
             -- backwards compatibility.
             apiError err400 PastHorizon $ T.unwords
                 [ "The transaction's validity interval is past the horizon"
-                , "of safe slot-to-time conversions."
+                , "of safe slot-to-time conversions: " <> t <> "."
```

Comment:
Ooh, nice! We have gotten some details back again. I think the exact `Text` comes from https://github.com/input-output-hk/cardano-wallet/blob/8e1b7239c090d3c110e8f24fb58396b5ec30327c/lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs#L1180 and is pretty long, so could we make this some `"\n\n Here are the full details:" <> t`  at the end?

Reply:
Thank you. I was trying to write a test that would return this error (I was also worried about the readability), as chasing the definition of the error down proved more difficult than expected. But triggering the error also proved difficult. I will move it to the end :th